magento-mpi/inventory-deprecated
# MC-29376: Stock indexer is executed twice after "products" import

### Description (*)
**Stock indexer is executed twice after "products" import**

Stock indexer "cataloginventory_stock" is executed (recent changes) right after import (before inventory import handler) as result, the indexer is executed twice.

The provided solution cancels the execution of stock indexer before inventory import handler.

### Manual testing scenarios (*)

PRECONDITION

- Stock Indexer is "update on save" mode

STEPS
1. Edit existing product or create one
2. Set quantity to 0 and disable the product
3. Export the product
4. Change the export file as following: `product_online = 1`; `qty = 100`; `is_in_stock = 1`
5. Import the edited file

ACTUAL
Stock indexer is executed twice

EXPECTED
Stock indexer is executed once

### Questions or comments


### Contribution checklist (*)
